### DISCUSSION TITLE
Documentation is not discoverable

### DISCUSSION THREAD

**Discussion by npaztwilio** [twilio-labs/paste#4192] [General] [answered]
Hey folks, just FYI. Your docs appear to not be getting indexed or something. 

Googling for Twilio Paste gives no results that lead to paste.twilio.design. Not sure what happened, I used to be able to find the docs pretty easily

**Comment by serifluous** [comment, marked as answer]
Hey @npaztwilio, thanks for reporting this! We're investigating right now in #4169 .

We'll reply here when we've got this fixed.

**Reply by npaztwilio** [reply]
Thanks! Didn't catch that other thread. I will close this as duplicate
